.travis.yml

```diff
@@ -2,14 +2,15 @@ language: go
 sudo: false
 
 go:
-  - master
-  - "1.10"
-  - "1.11"
+  - "1.12.x"
 
-install:
-  - go get -u golang.org/x/lint/golint
+env:
+  - GO111MODULE=on
+
+install: true
 
 script:
+  - GO111MODULE=auto go get -u golang.org/x/lint/golint
```

Comment:
only the `env` addition is needed. no need to modify the `install` and `script` keys.

https://github.com/carousell/listing/blob/ea4ede5a4e5c1328669508ec252c7fec7fe2b0a5/.travis.yml#L8-L15

Reply:
Because Travis build project under $GOPATH and make it as workspace, running the following will make changes to your go.mod.

It may break the dependency revision of your project if you used the same dependency as golint.

```
install:
  - go get -u golang.org/x/lint/golint
```

Reply:
makes sense